eva_submission/nextflow/variant_load.nf

```diff
@@ -119,3 +120,65 @@ process load_vcf {
     java -Xmx4G -jar $params.jar.eva_pipeline --spring.config.location=file:$params.eva_pipeline_props --parameters.path=$variant_load_properties
     """
 }
+
+
+accession_reports = Channel.fromPath(params.valid_vcfs)
+            .splitCsv(header:true)
+            .map{row -> tuple(file(row.vcf_file), row.db_name)}
```

Comment:
I don't think this `vcf_file` is the Accessioning report. This is the original valid VCF that was accessioned

Reply:
done